packages/eds-core-react/src/components/Button/Button.stories.tsx

```diff
@@ -154,22 +154,73 @@ FileUpload.decorators = [
   ),
 ]
 
-export const ProgressButton: Story<ButtonProps> = () => (
-  <>
-    <Button>
-      <Progress.Dots />
-    </Button>
-    <Button variant="ghost_icon">
-      <Progress.Dots color="primary" />
-    </Button>
-    <Button>
-      <Progress.Circular size={16} color="neutral" />
-    </Button>
-    <Button variant="ghost_icon">
-      <Progress.Circular size={16} color="primary" />
-    </Button>
-  </>
-)
+export const ProgressButton: Story<ButtonProps> = () => {
```

Comment:
Whups i accidentally comitted the progress button stuff. Remember to change back